### PR TITLE
650 course statistics(connects #650)

### PR DESCRIPTION
**Pull request by Pr0chin**

### CODE REVIEW

File: app/MyApp/index.html [hunk 1]
```diff
@@ -227,6 +227,8 @@
     <script src='app/views/CourseStepsView.js'></script>
     <script src='app/views/BecomeMemberForm.js'></script>
     <script src='app/views/CoursesStudentsProgress.js'></script>
+    <script src='app/views/CoursesStatistics.js'></script>
```
Comment (dogi) [inline app/MyApp/index.html]: like in https://github.com/open-learning-exchange/BeLL-Apps/commit/bed65a653d15896e9eec9ebdbf3765e58fadbafa every javascript include in `index.html` needs a special comment

**Review by dogi — CHANGES_REQUESTED** [review]

File: app/MyApp/index.html (hunk 1 above)
Comment (Pr0chin) [reply app/MyApp/index.html]: ok gotch ya ;) :D

File: app/MyApp/index.html [hunk 2]
```diff
@@ -227,8 +227,8 @@
     <script src='app/views/CourseStepsView.js'></script>
     <script src='app/views/BecomeMemberForm.js'></script>
     <script src='app/views/CoursesStudentsProgress.js'></script>
-    <script src='app/views/CoursesStatistics.js'></script>
-    <script src='app/views/CourseStepStatistics.js'></script>
+    <script src='app/views/CoursesStatistics.js'></script> <!-- app/views.js -->
```
Comment (dogi) [inline app/MyApp/index.html]: Good @Pr0chin ;)
but this is only half of the necessary changes
`app/views/CoursesStatistics.js` needs to be surrounded by double quotes `"`

**Review by dogi — CHANGES_REQUESTED** [review]

File: app/MyApp/index.html (hunk 2 above)
Comment (Pr0chin) [reply app/MyApp/index.html]: okz ;) :D @dogi

**Review by dogi — APPROVED** [review]

**Review by razu9861 — APPROVED** [review]

File: app/MyApp/app/Router.js [hunk 3]
```diff
@@ -2305,13 +2347,14 @@ $(function() {
...
                 $('.courseSearchResults_Bottom h2').append('<button id="manageOnCourseProgress" class="btn btn-success"  onclick = "document.location.href=\'#course/manage/' + cId + '\'">'+App.languageDict.attributes.Manage+'</button>')
+                $('.courseSearchResults_Bottom').append('<a id="CourseStatistics" class="btn btn-inverse"  href=\'#CourseStatistics/' + cId + '\'">'+App.languageDict.attributes.Course_Progress_Statistics+'</a>')
             }
             $('.courseSearchResults_Bottom').append('<p id="graph2title"style="text-align:center">'+App.languageDict.attributes.Individual_Member_Course_Progress+'</p>')
             App.$el.children('.body').append('<div id="detailView"><div id="graph2" class="flotHeight"></div><div id="choices" class="choice"></div></div><div id="birdEye"><div id="graph1" class="flotHeight"></div></div>')
             var allResults = new App.Collections.StepResultsbyCourse()
-            if (course.get('courseLeader').indexOf($.cookie('Member._id')) == -1  &&  roles.indexOf("Manager") == -1) {
+            /*if (course.get('courseLeader').indexOf($.cookie('Member._id')) == -1  &&  roles.indexOf("Manager") == -1) {
```
Comment (lmmrssa) [inline app/MyApp/app/Router.js]: not sure why this code is commented out so need some clarification

File: app/MyApp/app/collections/CourseAnswer.js [hunk 4]
```diff
@@ -3,7 +3,9 @@ $(function() {
     App.Collections.CourseAnswer = Backbone.Collection.extend({
 
         url: function() {
-            if (this.MemberID != "" && this.StepID != "" && this.pqattempts == "undefined") {
+            if ( this.StepID != "" && this.QuestionID!= "" /*&& this.pqattempts == "undefined" && this.MemberID == "undefined"*/) {
```
Comment (lmmrssa) [inline app/MyApp/app/collections/CourseAnswer.js]: need these commented out code from condition

File: init_docs/languages/English.txt [hunk 5]
```diff
@@ -310,6 +310,7 @@
   "Step_Method":"Step Method",
   "Step_Goals":"Step Goals",
   "Allowed_Errors":"Allowed Errors",
+  "Errors": "Errors",
```
Comment (lmmrssa) [inline init_docs/languages/English.txt]: need to remove these new lines in English.txt only needed in Text.json

**Review by lmmrssa — CHANGES_REQUESTED** [review]

**Review by Rupesh87 — APPROVED** [review]

File: init_docs/languages/English.txt (hunk 5 above)
Comment (Pr0chin) [reply init_docs/languages/English.txt]: removed from English.txt

File: app/MyApp/app/collections/CourseAnswer.js (hunk 4 above)
Comment (Pr0chin) [reply app/MyApp/app/collections/CourseAnswer.js]: Did it :D

File: app/MyApp/app/Router.js (hunk 3 above)
Comment (Pr0chin) [reply app/MyApp/app/Router.js]: corrected

**Review by dogi — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request closed without merging**